dashboard/app/views/maker/setup.html.haml

```diff
@@ -36,45 +36,10 @@
 -# If in the micro:bit experiment, display both the CP and MB board descriptions
 - if request.params['enableExperiments'] === 'microbit'
   .maker-setup
-    :markdown
-      # CS Discoveries Maker Toolkit IT Setup
-    .board-descriptions
-      .circuit-playground-description
```

Comment:
we will want to make sure to delete any associated styles as well (e.g., any styles associated with the classes 'board-descriptions' or 'circuit-playground-description')

Reply:
Would we want to apply those styles in the React file?

Reply:
yes, but we'll want to reimplement them in the react file. this is so that the backend/server code isn't responsible for styles used by the frontend

Reply:
I wanted to make sure all the styles are transferred to the React file. If things change shape and size, eyes test will fail.

Reply:
Styles are preserved